iotfunctions/anomaly.py

```diff
@@ -475,6 +514,10 @@ def execute(self, df):
                 else:
                     n_clus = 20
 
+                n_clus = np.minimum(n_clus, slices.size // 2)
```

Comment:
n_clus means the number of clusters? full variable name can give more information than an abbreviation.

Reply:
will be renamed to n_cluster